### DISCUSSION TITLE
Wrong icon for swiss-franc (or wrong naming)?

### DISCUSSION THREAD

**Discussion by mitosch** [lucide-icons/lucide#1658] [General]
Hi There

First, let me thank you for the great project. The icons just look nice, are easy to use and feel very understandable.

I've a question about the icon `swiss-franc`, which represents the unicode character U+20A3 (₣):

Maybe it has a reason to be named as swiss-franc?

Actually, it is the currency symbol for the old French Franc (a proposal in 1988, but was never introduced, [reference](https://en.wikipedia.org/wiki/French_franc#cite_note-5)). We (swiss people) do not have any symbol and just use the abbreviation (CHF, Fr. and sometimes sFr.).

I know, removing an icon can cause problems for users. But on the other hand, if a designer/developer thinks to use it for the swiss currency in their project, it could lead into a misunderstanding.

From my point of view, I'd deprecate and remove it later, because it is a symbol of an old, unused currency.

Cheers,

Mischa

**Comment by ericfennis** [comment]
Thanks for your kind words!

Oh good to know, I think we didn't dig into this that much. So our fault. 
We can rename icons, an alias will created to make everything compatible with existing projects that use Lucide.

Before we remove/deprecate this one, do you know if African countries that use Franc currency also use this symbol?

**Comment by mitosch** [comment]
I've made a short research and I'd say no. According to the linked Wikipedia article, the african franc is using the symbol F.CFA:

https://en.wikipedia.org/wiki/Central_African_CFA_franc

So, you can remove/deprecate the swiss-franc icon.
